core/tests/protractor_utils/forms.js

```diff
@@ -385,7 +385,10 @@ var RichTextChecker = function(arrayOfElems, arrayOfTexts, fullText) {
 
   var _readFormattedText = function(text, tagName) {
     expect(arrayOfElems[arrayPointer].getTagName()).toBe(tagName);
-    expect(arrayOfElems[arrayPointer].getInnerHtml()).toBe(text);
+    expect(
+      browser.executeScript(
```

Comment:
Here and in general.js, why not getAttribute('innerHTML')? (Seems less convoluted?)

http://stackoverflow.com/questions/39275627/what-should-we-use-in-place-of-getinnerhtml-and-getoutterhtml-in-selenium